packages/storage/package.json

```diff
@@ -30,10 +30,12 @@
     "lodash.throttle": "^4.1.1"
   },
   "peerDependencies": {
+    "@deephaven/filters": "^0.11.7",
     "@deephaven/log": "^0.11.2",
     "react": "^17.0.0"
   },
   "devDependencies": {
+    "@deephaven/filters": "file:../filters",
```

Comment:
I think the file path is fine for dev deps though. Since you would have the entire monorepo downloaded for development, but not for installing as a normal dep in a project

Reply:
See my above comment - it is fine for `devDependencies` and `dependencies`, does not appear to be fine for `peerDependencies` (which we should probably remove from most of our packages anyway.